sdk/kubeflow/trainer/types/types.py

```diff
@@ -50,37 +50,25 @@ class TrainJob:
     status: Optional[str] = "Unknown"
 
 
-# Configuration for the Lora to configure parameter efficient fine-tuning.
+# Configuration for the custom trainer.
 @dataclass
-class LoraConfig:
-    r: Optional[int] = field(
-        default=None, metadata={"help": "Lora attention dimension"}
-    )
-    lora_alpha: Optional[int] = field(default=None, metadata={"help": "Lora alpha"})
-    lora_dropout: Optional[float] = field(
-        default=None, metadata={"help": "Lora dropout"}
-    )
-
-
-@dataclass
-class FineTuningConfig:
-    # TODO (andreyvelich): Add more configs once we support them, e.g. QLoRA.
-    peft_config: Optional[LoraConfig] = None
-
-
-# Configuration for the Trainer.
-# TODO (andreyvelich): Discuss what values should be on the Trainer.
-@dataclass
-class Trainer:
-    """Trainer configuration.
-    TODO: Add the description
+class CustomTrainer:
+    """Custom Trainer configuration. Configure the self-contained function
+        that encapsulates the entire model training process.
+
+    Fields:
+        func: The function that encapsulates the entire model training process.
+        func_args: The arguments to pass to the function.
```

Comment:
We might want to modify it once we agreed on how to better allow user to pass custom parameters to with `CustomTrainer()`: https://github.com/kubeflow/trainer/pull/2384
cc @astefanutti